internal/cmd/builds/get.go

```diff
@@ -0,0 +1,81 @@
+package builds
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/saucelabs/saucectl/internal/build"
+	cmds "github.com/saucelabs/saucectl/internal/cmd"
+	"github.com/saucelabs/saucectl/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+func GetCommand() *cobra.Command {
+	var out string
+	var byJob bool
+	var buildSource string
+
+	cmd := &cobra.Command{
+		Use:          "get",
+		Short:        "Get build by build or job ID",
+		SilenceUsage: true,
+		Args: func(_ *cobra.Command, args []string) error {
+			if len(args) == 0 || args[0] == "" {
+				return errors.New("no build specified")
+			}
+			return nil
+		},
+		PreRun: func(cmd *cobra.Command, _ []string) {
+			tracker := usage.DefaultClient
+
+			go func() {
+				tracker.Collect(
+					cmds.FullName(cmd),
+					usage.Flags(cmd.Flags()),
+				)
+				_ = tracker.Close()
+			}()
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if out != JSONOutput && out != TextOutput {
+				return errors.New("unknown output format")
+			}
+
+			src := build.Source(buildSource)
+			if src != build.SourceRDC && src != build.SourceVDC {
+				return errors.New("invalid build resource. Options: vdc, rdc")
+			}
+
+			return get(cmd.Context(), args[0], byJob, src, out)
+		},
+	}
+	flags := cmd.PersistentFlags()
+	flags.BoolVarP(&byJob, "job-id", "", false, "Find the build by providing a job ID instead of a build ID.")
+	flags.StringVarP(&out, "out", "o", "text", "Output format to the console. Options: text, json.")
+	flags.StringVar(&buildSource, "source", "", "Build source from saucelabs. Options: vdc, rdc.")
```

Comment:
In `list.go`, you add `"Required."` to the end of the message. Is this required as well? Do you think it would be a good idea to move common options to a separate file for consistency?

Reply:
Adding the required notice here may be beneficial. I'll see what moving these to a common export would look like.

Reply:
I think this is resolved by nature of moving them to required positional arguments